code/game/objects/items/weapons/tools.dm

```diff
@@ -397,6 +397,11 @@
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
 		var/obj/item/organ/external/S = H.bodyparts_by_name[user.zone_sel.selecting]
+
+		if(H.bleed_rate > 1 && H.isSynthetic())

```

Comment:
This isn't going to do anything, the vast majority of the time; I'd just remove it; as soon as the next `Life` runs, it'll be back; it also means message spam.